examples/lx-instance/outputs.tf

```diff
@@ -1,16 +1,20 @@
 #Outputs from the Linux Instance Terraform module.
 
-output "watchmaker-lx-instance-stack-id" {
-  description = "This output provides the CloudFormation stack id of the Watchmaker Linux instance"
-  value       = "${module.lx-instance.watchmaker-lx-instance-stack-id}"
+output "watchmaker-lx-instance" {
+  description = "CloudFormation stack object for watchmaker-lx-instance"
+  value       = module.lx-instance.watchmaker-lx-instance
 }
 
 output "watchmaker-lx-instance-id" {
   description = "This output provides the Instance ID"
-  value       = "${module.lx-instance.watchmaker-lx-instance-id}"
+  value       = module.lx-instance.watchmaker-lx-instance.id
```

Comment:
can remove this output

Reply:
okay, no problem. was just giving as an example of how to use the rich object.